source/05-EL/06.ptx

```diff
@@ -276,6 +276,43 @@
         </answer>
       </task>
     </activity>
+
+    <activity xml:id="solving-log-equations-with-different-bases">
+      <introduction>
+        <p>
+          In some cases, you will get equations with logs of different bases. Apply properties of logarithms to solve the following logarithmic equations.
+        </p>
+      </introduction>
+      <task>
+        <statement>
+          <p>
+            <m>\log_3(x-6)=\log_9x</m>
+            <hint>
+              <p>
+                Use the change of base formula to rewrite <m>\log_9x</m> so that it has a base of <m>3</m>.
+              </p>
+            </hint>
+          </p>
+        </statement>
+        <answer>
```

Comment:
```suggestion
          </p>
        </statement>
        <hint>
              <p>
                Use the change of base formula to rewrite <m>\log_9x</m> so that it has a base of <m>3</m>.
              </p>
        </hint>
        <answer>
```

Reply:
Hints should be outside of statements